src/server/Completions/AttributeCompletionFactory.ts

```diff
@@ -110,4 +111,22 @@ export default class AureliaAttributeCompletionFactory extends BaseAttributeComp
   private propertyIsBindable(propertyDeclaration: ts.PropertyDeclaration): boolean {
     return propertyDeclaration.decorators?.some(decorator => { return decorator.getText().includes("@bindable"); });
   }
+
+  /**
+   * Fetches the equivalent component name based on the given ts.SourceFile
+   *
+   * @param sourceFile - The source file to map a component name to
+   */
+  private getElementNameFromSourceFile(sourceFile: ts.SourceFile): string {
+    return Path.basename(sourceFile.fileName).replace(/\.(ts|js|html)$/, '').split(/(?=[A-Z])/).map(s => s.toLowerCase()).join('-');
```

Comment:
```suggestion
    return Path.basename(sourceFile.fileName)
      .replace(/\.(ts|js|html)$/, '')
      .split(/(?=[A-Z])/)
      .map(s => s.toLowerCase())
      .join('-');
```

Reply:
the split.map.join looks like 'to-kebak-case'?

Reply:
It is, I reused the the way the `elementName` is resolved in `ProcessFile.ts` to ensure that the new solution hits the same nodes as the old.